vn/game/script.rpy

```diff
@@ -181,61 +134,44 @@ label start:
         zoom 0.6
     with Dissolve(.25)
 
-    priest_speech """
-        The world must be a very desolate place for them;
-        for them, literally nothing is distinguishable from shit.
-        How miserable indeed.
-    """
+    menu: 
 
-    show consequentialist:
-        xalign 0.85
-        yalign 0.70
-        zoom 0.25
-    with Dissolve(.25)
+        "There's nothing for it..." 
 
-    id_speech "Very"
+        "Walk forward.": 
+            "I inch forward apprehensively, my staff raised in front."
+            jump walk_forward
 
-    hide consequentialist
-    with Dissolve(.25)
+    label walk_forward: 
+        if menu_flag:
+            "From the sound I made earlier, I guessed I was in an open space, but no-"
 
-    show blindpriest at center:
+        else:
+            "Not for long, as I was stopped in my tracks by something tall, thin, metallic -

```

Comment:
I think you need the `"""` syntax for multiline comments. Might be wrong about this so ignore me if it works,

Reply:
Ah no worries, I'll amend and push it again

Reply:
Yeah you can just add a new commit and push to the same branch. It should update the PR automagically